src/iarc7_vision/roomba_blob_vision_node.py

```diff
@@ -344,7 +352,13 @@ def __init__(self, file_path):
         """
         :param file_path: path to video file
         """
-        super(VideoProcessor, self).__init__()
+        # Verify that the correct version of OpenCV is in use
+        if cv2.__version__[:3] != "2.4":
+            print("You are using the incorrect version of OpenCV. You " +
+                         "should be using version 2.4, but you are using " +
```

Comment:
They are in different classes. The CameraProcessor class relies on ROS, so it logs the error to the rospy output. The VideoProcessor class does not depend on ROS, so it logs the error to standard output.